test/nools/task-emitter.spec.js

```diff
@@ -95,6 +95,101 @@ describe('task-emitter', () => {
         });
       });
 
+      it('task priority is not required', () => {
+        // given
+        const config = {
+          c: personWithReports(aReport()),
+          targets: [],
+          tasks: [ aPersonBasedTask() ],
+        };
+        delete config.tasks[0].priority
+
+        // when
+        const { emitted } = runNoolsLib(config);
+
+        // then
+        expect(emitted).to.have.length(2);
+        expect(emitted[0]).to.not.include({
+          priority: 'high',
+          priorityLabel: [ { locale: 'en', label: 'High Priority' } ]
+        });
+      });
+
+      it('task priority is string set from object', () => {
+        // given
+        const config = {
+          c: personWithReports(aReport()),
+          targets: [],
+          tasks: [ aPersonBasedTask() ],
+        };
+        config.tasks[0].priority = function(c, r, e, d) {
```

Comment:
It looks like this function is failing the eslint test. Removing them would validate that the code works even when these parameters are not passed.